File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        dict1 =  defaultdict(int)
        for i in range(len(nums)):
            dict1[nums[i]] +=1;
        
        sorted_keys = sorted(dict1, key=dict1.get, reverse=True)
        ans=[]
        for i in range(k):
            ans.append(sorted_keys[i])
        
        return ans

File: Data Structures & Algorithms/counting-bits/submission-0.cpp
class Solution {
public:
    vector<int> countBits(int n) {
        vector<int> ans(n+1,0);
        for(int i=0; i<=n; i++){
            int count=0;
            for(int j=0; j<32; j++){
                if((1<<j) & i){
                    count++;
                }
            }
            ans[i]=count;
        }
        return ans;
    }
};
